### PR TITLE
Add convert to raw Flux option

### PR DESCRIPTION
**Pull request by noramullen1**
Addresses #2477

### CODE REVIEW

File: content/influxdb/cloud/notebooks/create-notebook.md [hunk 1]
```diff
@@ -23,42 +23,21 @@ This guide walks through the basics of creating a notebook. For specific example
...
-### Presentation mode
-Toggle the **Presentation** option to enter presentation mode. Use presentation mode to view notebooks in full screen, hiding the left and top navigation menus so only the cells appear. This mode might be helpful, for example, for stationary screens dedicated to monitoring visualizations.
-
-### Time range
-Select from the options in the dropdown list or select **Custom Time Range** to enter a custom time range with precision up to nanoseconds.
-
-### Timezone
-Click the timezone dropdown list to select a timezone to use for the dashboard. Select either the local time (default) or UTC.
+    {{< nav-icon "notebooks" >}}
+2. Click the log icon under the cells you want to view as Flux.
```
Comment (noramullen1) [inline content/influxdb/cloud/notebooks/create-notebook.md:41]: @drdelambre and @mnorriswalker -- Is this icon a log? It's not clear to me what it's supposed to be or what it has to do with Flux.

**Review by sanderson — APPROVED** [review]

File: content/influxdb/cloud/notebooks/create-notebook.md [hunk 2]
```diff
@@ -23,42 +23,21 @@ This guide walks through the basics of creating a notebook. For specific example
...
 If your cell contains a custom script that uses any output function to write data to InfluxDB (the `to()` function) or send it to a 3rd party service, clicking **Preview** will write data.
     {{% /warn %}}
 7. Click the dropdown menu next to **Preview** and select **Run** to show the results of each cell and write it to the specified output bucket.
-7. Click the eye icon to hide a cell.
+8. Click the eye icon to hide a cell.
 
-## Cell types
-Notebooks are comprised of different cells. Add one or more of the following cell types to your notebook:
+## View and edit Flux script in a notebook
+Convert your notebook cells into raw Flux script to view and edit it as code.
```
Comment (kelseiv) [inline content/influxdb/cloud/notebooks/create-notebook.md:36]:
```suggestion
Convert your notebook cells into raw Flux script to view and edit the code.
```

**Review by kelseiv — APPROVED** [review]

**Comment by noramullen1** [comment]
@drdelambre and @mnorriswalker -- checking in on the status of this feature.

**Comment by noramullen1** [comment]
@kelseiv @drdelambre Updated this to match current feature.

File: content/influxdb/cloud/notebooks/create-notebook.md [hunk 3]
```diff
@@ -77,27 +78,37 @@ Select one of the following pass-through cell-types:
...
+
+
+### View and edit Flux script in a cell
+Convert your notebook cells into raw Flux script to view and edit the code. Conversion to Flux is available for the following cell types:
+  - **Inputs: Metric Selector**
+  - **Transform: Column Editor** and **Downsample**
+
+1. Click the overflow menu icon under the cells you want to view as Flux, then select **Convert to |> Flux**. You won't be able to undo this step.
+2. A Flux cell appears with underlying script for the selected cell.
+3. View and edit the Flux script as needed.
```
Comment (kelseiv) [inline content/influxdb/cloud/notebooks/create-notebook.md:114]:
```suggestion
    A Flux cell appears with underlying script for the selected cell.
3. View and edit the Flux script as needed.
```

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/cloud/notebooks/create-notebook.md (hunk 3 above)
Comment (kelseiv) [reply content/influxdb/cloud/notebooks/create-notebook.md:114]: @noramullen1 looks good to me.

File: content/influxdb/cloud/notebooks/create-notebook.md (hunk 2 above)
Comment (kelseiv) [reply content/influxdb/cloud/notebooks/create-notebook.md]: @mluu12 let's add this one to our August Cloud updates

File: content/influxdb/cloud/notebooks/create-notebook.md [hunk 4]
```diff
@@ -77,27 +78,37 @@ Select one of the following pass-through cell-types:
...
+1. Select a bucket.
+2. Click **Preview** to see what would be written to the bucket without commiting, or click **Run** in the upper left to write, or select **Export as Task** to schedule your output as a task.
 
 #### Schedule
 
-1. Enter a time and an offset to schedule the task. 
-2. Click **Export as Task** to save. 
+1. Enter a time and an offset to schedule the task.
+2. Click **Export as Task** to save.
+
```
Comment (kelseiv) [inline content/influxdb/cloud/notebooks/create-notebook.md:105]:
```suggestion
```

File: content/influxdb/cloud/notebooks/create-notebook.md [hunk 5]
```diff
@@ -77,27 +78,37 @@ Select one of the following pass-through cell-types:
...
+
+### View and edit Flux script in a cell
+Convert your notebook cells into raw Flux script to view and edit the code. Conversion to Flux is available for the following cell types:
+  - **Inputs: Metric Selector**
+  - **Transform: Column Editor** and **Downsample**
+
+1. Click the overflow menu icon under the cells you want to view as Flux, then select **Convert to |> Flux**. You won't be able to undo this step.
```
Comment (kelseiv) [inline content/influxdb/cloud/notebooks/create-notebook.md:112]:
```suggestion
1. Click the overflow menu icon in the cell you want to view as Flux, then select **Convert to |> Flux**. You won't be able to undo this step.
```

**Pull request merged**